### PR TITLE
remove queries to user database, use user from known locations

### PR DESCRIPTION
**Pull request by wow-such-code**
Replaces unnecessary db connection and uses the sample tracking service location objects instead.

### CODE REVIEW

**Review by KochTobi — APPROVED** [review]
LGTM.
You do not need to provide a changelog entry. This will be generated automatically by your PR title.

File: src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy [hunk 1]
```diff
@@ -30,172 +30,175 @@ import static java.time.ZoneOffset.UTC
...
-  }
-
-  @Override
-  void updateSampleLocation(String sampleCode, Location location, String status, Instant timestamp, Person responsiblePerson) throws SampleUpdateException {
-    String locationJson = DtoMapper.createJsonFromLocationWithStatus(location, status, responsiblePerson, timestamp)
-    HttpResponse<String> response = requestSampleUpdate(createSampleUpdateURI(sampleCode), locationJson)
-    if (response.statusCode() != 200) {
-      throw new SampleUpdateException("Could not update $sampleCode to ${location.getLabel()} - ${response.statusCode()} : ${response.headers()}: ${response.body()}")
-    }
-  }
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy:66]: why was this removed?

**Review by KochTobi — CHANGES_REQUESTED** [review]

File: src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy (hunk 1 above)
Comment (wow-such-code) [reply src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy:66]: This was a duplicate due to the merge, the correct method does the same without responsible person object (it's taken from the location itself)
